README.md

```diff
@@ -80,7 +80,7 @@ It correctly bundles React in production mode and optimizes the build for the be
 The build is minified and the filenames include the hashes.<br>
 Your app is ready to be deployed!
 
-## How Do I?...
+## How Do I...?
```

Comment:
Hey can you change the `## How Do I...?` to `## How Do I…?`

The second uses the ellipses character rather than 3 dots :) 

You can just copy the character from this comment.